### PR TITLE
YUNIKORN-746: Simplify the REST API document

### PR DESCRIPTION
**Pull request by manirajv06**
Doc changes to clearly explain the Old and Newer version API's use.

### CODE REVIEW

**Comment by yangwwei** [comment]
hi @manirajv06  thanks for working on this. Can we remove the "Deprecated" and "Newer version" in the doc, and merge them into one section? For queus/apps/nodes, we should only have one section each, and the old/new API can all be explained in each single section.

**Comment by manirajv06** [comment]
Taken care. Pls check.

File: docs/api/scheduler.md [hunk 1]
```diff
@@ -88,7 +88,9 @@ Displays general information about the partition like name, state, capacity, use
 }
 ```
 
-## Queues (Newer Version)
+## Queues
+
+### Single Partition
```
Comment (yangwwei) [inline docs/api/scheduler.md:93]: Partition queues

File: docs/api/scheduler.md [hunk 2]
```diff
@@ -162,7 +164,7 @@ For the default queue hierarchy (only `root.default` leaf queue exists) a simila
 }
 ```
 
-## Queues
+### All Partitions
```
Comment (yangwwei) [inline docs/api/scheduler.md:167]: All queues

File: docs/api/scheduler.md [hunk 3]
```diff
@@ -217,7 +219,9 @@ For the default queue hierarchy (only `root.default` leaf queue exists) a simila
 }
 ```
 
-## Applications (Newer Version)
+## Applications
+
+### Single Partition
```
Comment (yangwwei) [inline docs/api/scheduler.md:224]: Queue applications

File: docs/api/scheduler.md [hunk 4]
```diff
@@ -319,7 +323,7 @@ In the example below there are three allocations belonging to two applications.
 }
 ```
 
-## Applications
+### All Partitions
```
Comment (yangwwei) [inline docs/api/scheduler.md:326]: All applications

File: docs/api/scheduler.md [hunk 5]
```diff
@@ -401,7 +405,9 @@ In the example below there are three allocations belonging to two applications.
 ]
 ```
 
-## Nodes (Newer Version)
+## Nodes
+
+### Single Partition
```
Comment (yangwwei) [inline docs/api/scheduler.md:410]: Partition nodes

File: docs/api/scheduler.md [hunk 6]
```diff
@@ -508,7 +514,7 @@ Here you can see an example response from a 2-node cluster having 3 allocations.
 }
 ```
 
-## Nodes
+### All Partitions
```
Comment (yangwwei) [inline docs/api/scheduler.md:517]: All nodes

**Review by yangwwei — CHANGES_REQUESTED** [review]
Thanks, it looks much better. I just added some minor comments to request the change of sub-titles, hope that makes sense.

**Review by yangwwei — APPROVED** [review]
LGTM

**Pull request merged**